logstash.go

```diff
@@ -54,17 +35,32 @@ func NewLogstashAdapter(route *router.Route) (router.LogAdapter, error) {
 	}
 
 	return &LogstashAdapter{
-		route: route,
-		conn:  conn,
+		route:         route,
+		conn:          conn,
+		containerTags: make(map[string][]string),
 	}, nil
 }
 
+// Get container tags configured with the environment variable LOGSTASH_TAGS
+func GetContainerTags(c *docker.Container, a *LogstashAdapter) []string {
+	var tags []string
+	if val, ok := a.containerTags[c.ID]; ok {
+		tags = val
+	} else {
+		for _, e := range c.Config.Env {
+			if strings.HasPrefix(e, "LOGSTASH_TAGS=") {
+				tags = strings.Split(strings.TrimPrefix(e, "LOGSTASH_TAGS="), ",")
+				break
+			}
+		}
+		a.containerTags[c.ID] = tags
+	}
+	return tags
```

Comment:
I still need to add the key here, else for a container with no tags it always loop through all the environment variables... What do you think of declaring the variable before the for loop and keep the break statement?